### ISSUE TITLE
Lists not rendering on the challenge page correctly

### ISSUE THREAD

**Issue by jmgasper** [topcoder-platform/work-manager#915]
I'm not sure if this is a community app issue or an issue with work manager, but lists aren't rendering properly.

### What I see in WM

<img width="1497" alt="Screen Shot 2020-11-27 at 9 09 33 am" src="https://user-images.githubusercontent.com/21790/100394222-6d2cab80-3090-11eb-97ca-45154d55781e.png">

### What I see on community app

<img width="734" alt="Screen Shot 2020-11-27 at 9 09 29 am" src="https://user-images.githubusercontent.com/21790/100394225-6f8f0580-3090-11eb-89f2-58455cf50cf8.png">

**Comment by vikasrohit** [comment]
@acshields I don't think it belongs to WM, it seems the issue is with renderer in the Community side.

**Comment by acshields** [comment]
@vikasrohit - whoops! you are right. I will send this one over to Oanh for her to prioritize.
